main.cpp

```diff
@@ -52,85 +52,85 @@ arm.rotateCounterClockwise();           // 4) demonstrates CamelCase
 
 
 //  1)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
- 
+//  Noun: dog
+//  action 1: the dog barks
+dog.bark();
+//  action 2: the dog fetches
+dog.fetch();
+//  action 3: the dog growls
+dog.growl();
 //  2)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
- 
+//  Noun: plane
+//  action 1: the plane flies
+plane.fly();
+//  action 2: the plane lands
+plane.land();
+//  action 3: the plane takes off
+plane.takeOff();
 //  3)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
- 
+//  Noun: athlete
+//  action 1: the athlete trains
+athlete.train();
+//  action 2: the athlete wins
+athlete.lose();
+//  action 3: the athlete loses
+athlete.win();
 //  4)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
- 
+//  Noun: bear
+//  action 1: the bear eats
+bear.eat();
+//  action 2: the bear hibernates
+bear.hibernate();
+//  action 3: the bear scares people
+bear.scare();
 //  5)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
- 
+//  Noun: teacher
+//  action 1: the teacher teaches
+teacher.teach();
+//  action 2: the teacher grades
+teacher.grade();
+//  action 3: the teacher inspires
+teacher.inspire();
 //  6)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
- 
+//  Noun: bird
+//  action 1: the bird flies
+bird.fly();
+//  action 2: the bird chirps
+bird.chirp();
+//  action 3: the bird lays an egg
+bird.layEgg();
 //  7)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
- 
+//  Noun: computer
+//  action 1: the computer loads
+computer.load();
+//  action 2: the computer compiles code
+computer.compile();
+//  action 3: the computer computes
+computer.compute();

```

Comment:
As explained in the instructions, here is the obligatory request for some changes to get you familiar with the Pull Request Review process.
**REPLACE** the highlighted noun/action as follows:
- [ ] come up with a new noun and 3 actions
- [ ] pick a noun that needs `camelCase` for the name
- [ ] pick actions that need `camelCase` for their name.
**Do not** create an 11th noun.